src/components/structures/LoggedInView.tsx

```diff
@@ -694,8 +695,10 @@ class LoggedInView extends React.Component<IProps, IState> {
                 >
                     <ToastContainer />
                     <div className={bodyClasses}>
-                        <div className='mx_LeftPanel_wrapper'>
-                            { SettingsStore.getValue('TagPanel.enableTagPanel') &&
+                        <div className='mx_LeftPanel_outerWrapper'>
```

Comment:
Weird diff, change is adding this extra wrapper `mx_LeftPanel_outerWrapper ` and `LeftPanelLiveShareWarning`

Reply:
looks like the block for `mx_GroupFilterPanelContainer` didn't get indented, which confused the diff a bit. Not that diffs to jsx are ever clear... 😅